### ISSUE TITLE
unable to override CSS

### ISSUE THREAD

**Issue by TravisDunlop** [RinteRface/bs4Dash#114]
Hello! I would like to override CSS used by AdminLTE3 and am having some trouble. 

As an example, I am trying to enable scrolling in the right controlbar.  I found [this Issue in the AdminLTE repo](https://github.com/ColorlibHQ/AdminLTE/issues/1143) and am trying to implement their solution.  Here is what I tried:

```R
library(shiny)
library(bs4Dash)

shiny::shinyApp(
  ui = bs4DashPage(
    navbar = bs4DashNavbar(),
    sidebar = bs4DashSidebar(),
    controlbar = bs4DashControlbar(
      'this is some',
      paste(rep('really', 1000), collapse = ' '),
      'long content'
    ),
    footer = bs4DashFooter(),
    title = "test",
    body = bs4DashBody(
      tags$style(HTML(".control-sidebar { .tab-content { overflow-y: scroll; } }")),
      'Hello, World!'
    )
  ),
  server = function(input, output, session) { }
)
```

I am not sure if this doesn't work because CSS can't be overridden in this manner, I messed up the CSS, or both.

I'm framing this issue as a more broad ('how do I override CSS?') instead of narrow ('how do I scroll in the right controlbar?') because I have run into a couple other edge cases like this (where there is a CSS solution for AdminLTE but I'm unsure how to implement) and would like to be able to solve these as well.

By the way, thank you for this repo! I've been really enjoying developing using bs4Dash!  😊

**Comment by mattfemia** [comment]
@TravisDunlop Hey you might have resolved this already, but I had trouble with the CSS a while ago too. I got yours to work by removing the {.tab-content}:

`tags$style(HTML(".control-sidebar { overflow-y: scroll } "))`

So I think it may have been the CSS selectors you were using. I couldn't find a tab-content class in the control-bar of your reprex, but if you need it, you'll need to change the syntax. Check out https://www.w3schools.com/cssref/css_selectors.asp for specifics on where it's nested in your file.

Also to answer your broad question: if you use the 'www' directory format that shiny implements and create an external CSS file in the www directory then you can do this:

`
bs4DashPage(
...
body = bs4DashBody(
tags$head(includeCSS("www/yourFile.css"))
...
)
`

It will render 'yourFile.css' after all of the shiny/bs4Dash dependencies so that you can do custom css-overrides. Ended up really helping me with all shiny projects going forward, so hopefully for you too!

**Comment by DivadNojnarg** [comment]
Fixed in latest bs4Dash development version v2.0.0.
